remodel/models.py

```diff
@@ -76,12 +76,13 @@ def save(self):
             result = (r.table(self._table).get(id_).replace(r.row
                         .without(r.row.keys().difference(list(fields_dict.keys())))
                         .merge(fields_dict), return_changes='always').run())
-
-        except KeyError:
+            if result['errors'] > 0:
```

Comment:
What you are trying to fix here?

Reply:
I've reverted this change, since it is not entirely clear to me what the issue is (and it's also not related to the PRs title). Feel free to open an issue about this and I would gladly discuss it with you!